app/main.jsx

```diff
@@ -15,22 +15,27 @@ const Root = connect(
   ({ auth }) => ({ user: auth })
 ) (
   ({ user, children }) =>
-    <div>
-      <nav>
-        {user ? <WhoAmI/> : <div><Login/> <GoogleLogin/> </div>}
-      </nav> 
-      {children}
+
+    <div id="navbar">
+      <h1 id="title">Poppin' Potions</h1>
+      <div id="nav">
+        <nav>
+          {user ? <WhoAmI /> : <div><Login /> <GoogleLogin /> <Navbar /></div>}
+        </nav>
+        {children}
+      </div>
     </div>
+
 )
 
 render (
   <Provider store={store}>
     <Router history={browserHistory}>
       <Route path="/" component={Root}>
-        <IndexRedirect to="/potions"/>
-        <Route path="/potions" component={Navbar}/>
+        <IndexRedirect to="/potions" />
```

Comment:
As with comment on earlier commit, we have to make sure our products component is still being rendered, as shown here: 

`  <Provider store={store}>
    <Router history={browserHistory}>
      <Route path="/" component={ExampleApp} onEnter={onLoadProducts}>
        <IndexRedirect to="/home" />
        <Route path="/home" component={ProductsContainer} />
      </Route>
    </Router>
  </Provider>,
  document.getElementById('main')`